core/imageroot/var/lib/nethserver/cluster/actions/create-cluster/01generate_uuid

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import agent
+import uuid
+
+redis_client = agent.redis_connect(privileged=True)
+
+if redis_client.exists('cluster/uuid') != True:
+    redis_client.set('cluster/uuid', str(uuid.uuid4()))
```

Comment:
create-cluster is designed to run once. The existence check is useless because the validation step already does it.

I'd prefer to remove this step and run the .set() here

https://github.com/NethServer/ns8-core/blob/c71da2722c6d1574ad7042447bc9621dd80ae004/core/imageroot/var/lib/nethserver/cluster/actions/create-cluster/50update#L53

----
Just for the record the Redis [SETNX](https://redis.io/commands/setnx/) command might be useful in similar situations!